Modules/Loadable/Terminologies/Logic/vtkSlicerTerminologiesModuleLogic.h

```diff
@@ -269,6 +271,13 @@ class VTK_SLICER_TERMINOLOGIES_LOGIC_EXPORT vtkSlicerTerminologiesModuleLogic :
   /// Assemble human readable info string from a terminology entry, for example for tooltips
   static std::string GetInfoStringFromTerminologyEntry(vtkSlicerTerminologyEntry* entry);
 
+  /// Compare two terminology entries for equality.
+  /// \return True if the entries are equal, false otherwise.
+  bool AreSegmentTerminologyEntriesEqual(vtkSegment* segment1, vtkSegment* segment2);
+  bool AreTerminologyEntriesEqual(vtkSlicerTerminologyEntry* entry1, vtkSlicerTerminologyEntry* entry2);
+  bool AreTerminologyEntriesEqual(std::string terminologyEntry1, std::string terminologyEntry2);
+  bool AreCodedEntriesEqual(vtkCodedEntry* codedEntry1, vtkCodedEntry* codedEntry2);
```

Comment:
I was wondering if these could be static methods and maybe `AreCodedEntriesEqual` could go into vtkCodedEntry class, but then I realized that in the future the conversion may involve translation tables (that allows comparing of terms between different coding schemes). So, it makes sense to have these methods as non-static member functions of the logic. Maybe you could add this as a developer note (comment with `//` so that it is not included in the generated user documentation) so that we remember this for the future.